README.md

```diff
@@ -74,6 +79,13 @@ Simple test that checks whether a file or directory exists. This indicates it in
 ### get(path:  string):  Promise<string  |  Buffer>
 Gets a file as a string/Buffer.
 
+### activitiesGet(objectId: number | string, sort?: 'asc' | 'desc', limit?: number, sinceActivityId?: number):  Promise\<OcsActivity[]\>
```

Comment:
Why `activitiesGet` and not e.g. `getActivitiesFor` ?

Reply:
I would like a single entry point for all OCS requests.
If I start typing activities, or users/groups/whatever, I will see all the methods grouped up.
I'm open to suggestions.

Reply:
why not simply `getActivities`?